vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -19,6 +19,8 @@ Vizro-AI currently supports the following [OpenAI models](https://platform.opena
     - gpt-3.5-turbo-1106
     - gpt-3.5-turbo-0125
     - gpt-4o-2024-05-13
+    - gpt-4o-mini-2024-07-18
+    - gpt-4o-mini
```

Comment:
Can we also double check the other models, if they are still up-to-date?

Reply:
Just double checked, everything is up to date so far 👍